tests/pytype_test.py

```diff
@@ -65,27 +87,49 @@ def communicate(self):
 
 def pytype_test(args):
     try:
-        PytdRun(["-h"]).communicate()
+        BinaryRun(['pytd', '-h']).communicate()
     except OSError:
-        print("Cannot run pytd. Did you install pytype?")
+        print('Cannot run pytd. Did you install pytype?')
         return 0, 0
 
-    wanted = re.compile(r"stdlib/.*\.pyi$")
-    skipped = re.compile("(%s)$" % "|".join(load_blacklist()))
-    files = []
+    blacklist, parse_only = load_blacklist()
```

Comment:
"parse_only" is from the blacklist, too, so that variable name is a bit confusing.

Rename `blacklist` to `skip`?

Reply:
Done